### PR TITLE
Plurimath Formula to OMML conversion

### PR DESCRIPTION
**Pull request by suleman-uzair**
This PR updates in function class's conversion methods for OMML conversion.

### CODE REVIEW

**Review by HassanAkbar — APPROVED** [review]
Overall the PR looks good 👍 , just a minor code refactor comment.

File: lib/plurimath/math/function/binary_function.rb [hunk 1]
```diff
@@ -85,6 +81,10 @@ def empty_tag(wrapper_tag)
           r_tag << (Utility.ox_element("t", namespace: "m") << "&#8203;")
           wrapper_tag << r_tag
         end
+
+        def all_values_exist?
+          !(parameter_one.nil? || parameter_two.nil?)
```
[the remark this answers is not in the document]
Comment (ronaldtse) [reply lib/plurimath/math/function/binary_function.rb]: Thank you @HassanAkbar !

**Review by ronaldtse — APPROVED** [review]
Thank you @suleman-uzair !

**Comment by ronaldtse** [comment]
@suleman-uzair please help release a new version when ready. Thanks!

**Comment by suleman-uzair** [comment]
> @suleman-uzair please help release a new version when ready. Thanks!

@ronaldtse I have released the version 0.3.8.

**Pull request merged**